resources/_gen/assets/js/js/search.js_b4bb3e7b21b43e920b7c367ff787d383.content

```diff
@@ -27,7 +27,7 @@ limitations under the License.
                     }
 
                     var query = $(this).val();
-                    var searchPage = "https://knative.dev/search/?q=" + query;
+                    var searchPage = "http://localhost:1313/search/?q=" + query;
```

Comment:
Revert please (its from using `hugo server`). There is currently a workaround that we must perform in order to use the "extended version" of Hugo with our host, Netlify. Otherwise, the builds wont actually render all your SCSS changes (this workaround has recently been fixed on Netlify (and in the Docsy project). I have a pending TODO for switching us over to the supported Netlify environment).

1. delete both the `public` and `resources` folders from the root of your knative/docs clone
2. from the root, run `hugo` (runs a normal Hugo build)
3. add all the changes to this PR (you should have clean versions of the `public` and `resources` folders in root now)

Reply:
ok, I've built with the hugo command after deleting public & resources and pushed the results. Let me know if that works or there's anything else I need to do. 

Thanks,
Josh